chapters/untyped_pointers.md

```diff
@@ -0,0 +1,129 @@
+# Untyped Pointers
+
+The motivation for [SPV_KHR_untyped_pointers](https://github.khronos.org/SPIRV-Registry/extensions/KHR/SPV_KHR_untyped_pointers.html) is based on the recognition that explicit pointee types carried no real semantic value. Having a `int* ptr` really can just be a `void* ptr` as it can always be cast (with `OpBitcast`). The core idea of untyped pointers is not until you make the memory access (`OpLoad`, `OpStore`, etc) the pointer type is not needed. This matches the mental model of using things like C++ templates or unions.
```

Comment:
```suggestion
The motivation for [SPV_KHR_untyped_pointers](https://github.khronos.org/SPIRV-Registry/extensions/KHR/SPV_KHR_untyped_pointers.html) is based on the recognition that explicit pointee types carried no real semantic value. Having a `int* ptr` really can just be a `void* ptr` as it can always be cast (with `OpBitcast`). The core idea of untyped pointers is that until you make the memory access (`OpLoad`, `OpStore`, etc) the type is not needed. This matches the mental model of using things like C++ templates or unions.
```